Source/WPEProcess/Process.cpp

```diff
@@ -649,7 +649,22 @@ int main(int argc, char** argv)
             if (options.Group != nullptr) {
                 Core::ProcessCurrent().Group(string(options.Group));
             }
+
+            Core::Directory dataPath(options.DataPath.c_str());
+            dataPath.Permission((S_IRUSR | S_IWUSR | S_IRGRP | S_IWGRP));
+
+            Core::Directory persistentPath(options.PersistentPath.c_str());
+            persistentPath.Permission((S_IRUSR | S_IWUSR | S_IRGRP | S_IWGRP));
+
             if (options.User != nullptr) {
+                string group = Core::ProcessCurrent().Group();
```

Comment:
Also here, ther is something incorrect on the way it is working. It is the PLugins, responsibility to make sure that the rights are handled correctly. Now we are overruling the settings of a plugin to, by definition, allow a group user to access is. It could be that the plugin developer wants the spawned out-of-process plugin part not to write to the Persistent directory. If so, we know killed that feature :-) So I think the settings on any directory from the spawned process should never happen. 
Lets investigate the root-casue that lead to this change (Amazon rights ? ) and lets fix it in the Amazon Plugin. Talked already also to @bramoosterhuis , There is a CreatePersistentPath that probably requires right settings.

Reply:
changed data path settings to image creation from build system